.evergreen/config/generate-config.php

```diff
@@ -13,7 +14,9 @@
 ];
 $supportedPhpVersions = array_merge($modernPhpVersions, $legacyPhpVersions);
 
-$latestPhpVersion = max($supportedPhpVersions);
+// TODO: use max() once PHP 8.4 is stable
+//$latestPhpVersion = max($supportedPhpVersions);
+$latestPhpVersion = '8.3';
```

Comment:
Since PHP 8.4 isn't stable yet, I figured that the extra tests that run on the latest PHP version should continue to run on 8.3